### ISSUE TITLE
In guidelines, the links to techniques should have same name

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#533]
Now that the titles of key information sections are changed, we need to have the headings in the links to the techniques get updated as well.
